### DISCUSSION TITLE
Custom Option to resize window

### DISCUSSION THREAD

**Discussion by Ramalama2** [rxhanson/Rectangle#1058] [Ideas]
Hi, it would be really cool, if we had at least one option to resize a window to a specific resolution, or aspect ratio etc xD
Would be extremely handy for me for web development.

Another thing is, I bought magnet, but rectangle, hell...
You made a really amazing job with this app, basically it does everything magnet does and more... and all that for free....
you must be very motivated xD

Thank you a lot for this!

**Comment by rxhanson** [comment]
Thanks for the kind feedback. Rectangle is intentionally slim. [Rectangle Pro](https://rectangleapp.com/pro) (paid) has a custom shortcut builder where you can set up any size window (pixel or screen percentage) and select the position.

**Reply by Ramalama2** [reply]
oh i didn't knew about the pro xD
thanks for the info, thats perfect!
